app/views/internal/feedback_messages/index.html.erb

```diff
@@ -1,146 +1,26 @@
-      <div class="card-body" style="overflow: scroll; max-height: 500px;">
-        <% @vomits.each do |reaction| %>
-          <% next if reaction.reactable_type == "Article" && !reaction.reactable.published %>
-          <div class="d-flex justify-content-between">
-            <span>
-              🤢 <a href="<%= reaction.user.path %>">@<%= reaction.user.username %></a>
-            </span>
-            <span>
-              <strong><%= reaction.reactable_type %>:</strong>
-              <a href="<%= reaction.reactable.path %>"><%= reaction.reactable.title %></a>
-            </span>
-            <span>
-              <% if params[:status] == "Open" || params[:status].blank? %>
-                <%= form_for [:internal, reaction], html: { class: "d-inline" } do |f| %>
-                  <%= f.hidden_field :status, value: "confirmed" %>
-                  <%= f.submit "CONFIRMED", class: "btn btn-success btn-sm" %>
-                <% end %>
-                <%= form_for [:internal, reaction], html: { class: "d-inline" } do |f| %>
-                  <%= f.hidden_field :status, value: "invalid" %>
-                  <%= f.submit "INVALID", class: "btn btn-danger btn-sm" %>
-                <% end %>
-              <% end %>
-            </span>
-          </div>
-          <hr>
-        <% end %>
-      </div>
-    </div>
-  </div>
-</div>
-
-<%= search_form_for @q, url: internal_feedback_messages_path, class: "form-inline justify-content-end" do |f| %>
-
-  <%= f.label :reported_url_cont, "Reported URL", class: "sr-only" %>
-  <%= f.search_field :reported_url_cont, placeholder: "Reported URL", class: "form-control mx-3" %>
-
-  <%= f.label :reporter_username_cont, "Reporter", class: "sr-only" %>
-  <%= f.search_field :reporter_username_cont, placeholder: "Reporter", class: "form-control mx-3" %>
-
-  <%= f.select :status_eq,
-    options_for_select([
-      ["Open"],
-      ["Invalid"],
-      ["Resolved"],
-    ], @q.status_eq),
-    {},
-    class: "custom-select mx-3" %>
-
-  <%= f.submit "Search", class: "btn btn-secondary" %>
-<% end %>
-
-<div class="row mt-4 justify-content-center">
-  <%= content_tag :span, sort_link(@q, :reporter_username, "Reporter"), class: "mx-2" %>
-  <%= content_tag :span, sort_link(@q, :reported_url, "Reported URL"), class: "mx-2" %>
-  <%= content_tag :span, sort_link(@q, :created_at, "Creation Date"), class: "ml-2" %>
-</div>
-
-<%= paginate @feedback_messages %>
 
-<% @feedback_messages.each do |feedback_message| %>
-  <%= form_for [:internal, feedback_message] do |f| %>
-    <%= render "feedback_message", f: f, feedback_message: feedback_message %>
-  <% end %>
-<% end %>
 
-<%= paginate @feedback_messages %>
+<% if @view == "new-articles" %>
+  <%= render "new_articles" %>
+<% elsif @view == "suspicious-users" %>
+  <%= render "suspicious_users" %>
+<% else %>
+  <%= render "reports" if @view.blank? %>
+<% end %>
```

Comment:
these could just be rendered by the controller.

`/internal/reports/new-articles`
`/internal/reports/suspicious-users`
`/internal/reports/feedback-messages` or just `/internal/reports`